fix: broken links to SDK repo and remove references to guide (#488)
safe-global/safe-docs: pages/sdk/protocol-kit.mdx

# Protocol Kit

> The [Protocol Kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/protocol-kit) enables developers to interact with [Safe Smart Accounts](https://github.com/safe-global/safe-smart-account) using a TypeScript interface. This Kit can be used to create new Safe accounts, update the configuration of existing Safes, propose and execute transactions, among other features.

Reviewer: ```suggestion
The [Protocol Kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/protocol-kit) enables developers to interact with the [Safe Smart Account](https://github.com/safe-global/safe-smart-account) using a TypeScript interface. This Kit can be used to create new Safe accounts, update the configuration of existing ones, propose and execute transactions, among other features.
```

### Initialize the Protocol Kit

> The `SafeFactory` class allows the deployment of new Safe accounts while the `Safe` class represents an instance of a specific Safe Smart Account.

Reviewer: ```suggestion
The `SafeFactory` class allows the deployment of new Safe accounts while the `Safe` class represents an instance of a specific one.
```

### Conclusion

> In this quickstart, you learned how to create and deploy a Safe Smart Account and to propose and then execute a transaction from it.

Reviewer: ```suggestion
In this quickstart, you learned how to create and deploy a Safe Smart Account and to propose and then execute a transaction from it.
```
```suggestion
In this quickstart, you learned how to create and deploy a new Safe account and to propose and then execute a transaction from it.
```
